Sources/Subs-Db-mysql.php

```diff
@@ -414,7 +414,7 @@ function smf_db_query($identifier, $db_string, $db_values = array(), $connection
 		$old_pos = 0;
 		$pos = -1;
 		// Remove the string escape for better runtime
-		$db_string_1 = str_replace('\\\'', '', $db_string);
+		$db_string_1 = str_replace('\'\'','',$db_string);
```

Comment:
For the sake of easier reading, could you change this one to use double quotes around  the overall string, too?